Themes/package.json

```diff
@@ -0,0 +1,53 @@
+{
+  "name": "cpptools-themes",
+  "displayName": "C/C++ Themes",
+  "description": "UI Themes for C/C++ extension.",
+  "version": "0.24.1",
+  "publisher": "ms-vscode",
+  "preview": true,
```

Comment:
Does this token tell VS Code to preview the theme when the extension is installed?  If you set this to false, can we bundle it with our extension after all?

Reply:
This does not appear to be related to themes.

https://code.visualstudio.com/api/references/extension-manifest


preview |   | boolean | Sets the extension to be flagged as a Preview in the Marketplace.
-- | -- | -- | --

In emails with @aeschli , he seemed to imply this behavior is not currently optional.

Reply:
Ok.  Then we should set this to false since the theme extension will not be marked as "Preview" in the Marketplace.